HISTORY.md

```diff
@@ -2,9 +2,15 @@
 ## Unreleased
 ### Bug Fixes
 * If the primary's CURRENT file is missing or inaccessible, the secondary instance should not hang repeatedly trying to switch to a new MANIFEST. It should instead return the error code encountered while accessing the file.
+* Fix `GetSortedWalFiles()` may return IO error during WAL file deletion.
+
 ### New Features
 * Made the EventListener extend the Customizable class.
 * EventListeners that have a non-empty Name() and that are registered with the ObjectRegistry can now be serialized to/from the OPTIONS file.
+
+### Performance Improvements
+* Try to avoid updating DBOptions if `SetDBOptions()` does not change any option value.
```

Comment:
Does this need to be moved or is this an accidental merge issue?

Reply:
sorry for the confusion, it needs to move to correct my previous merge issue. I will create a separate minor PR for that.